crypto/aes/build.info

```diff
@@ -38,6 +38,8 @@ GENERATE[aesv8-armx.S]=asm/aesv8-armx.pl $(PERLASM_SCHEME)
 INCLUDE[aesv8-armx.o]=..
 GENERATE[vpaes-armv8.S]=asm/vpaes-armv8.pl $(PERLASM_SCHEME)
 
+GENERATE[aes-armv4.S]=asm/aes-armv4.pl $(PERLASM_SCHEME)
+GENERATE[bsaes-armv7.S]=asm/bsaes-armv7.pl $(PERLASM_SCHEME)
```

Comment:
I've got a debug log with `make -d` and put it on https://gist.github.com/shigeki/8ad0483a658e3602cc6a. 

There is a line 
```txt
 Found an implicit rule for 'crypto/aes/aes-armv4.s'
````
at https://gist.github.com/shigeki/8ad0483a658e3602cc6a#file-make_debug_log-txt-L6754.

 And finally it goes an error at the last as 
```txt
Finished prerequisites of target file 'crypto/aes/aes-armv4.S'.
           Must remake target 'crypto/aes/aes-armv4.S'.
CC="gcc" /usr/bin/perl crypto/aes/asm/aes-armv4.pl linux32 crypto/aes/aes-armv4.S
Putting child 0x199a978 (crypto/aes/aes-armv4.S) PID 397 on the chain.
Live child 0x199a978 (crypto/aes/aes-armv4.S) PID 397 
Reaping winning child 0x199a978 PID 397 
Removing child 0x199a978 PID 397 from chain.
           Successfully remade target file 'crypto/aes/aes-armv4.S'.
           Finished prerequisites of target file 'crypto/aes/aes-armv4.s'.
          Must remake target 'crypto/aes/aes-armv4.s'.
gcc -E  crypto/aes/aes-armv4.S > crypto/aes/aes-armv4.s
Putting child 0x199a978 (crypto/aes/aes-armv4.s) PID 401 on the chain.
Live child 0x199a978 (crypto/aes/aes-armv4.s) PID 401 
crypto/aes/aes-armv4.S:35:23: fatal error: arm_arch.h: No such file or directory
 # include "arm_arch.h"
````

It seems that the rule in `generatesrc` was not applied.